### ISSUE TITLE
Tutorial: Visual Object Recognition

### ISSUE THREAD

**Issue by moi90** [tensorflow/tensorflow#400]
When will the 'Visual Object Recognition' part of the tutorial be released?

**Comment by shlens** [comment]
The tutorial has been released.
https://www.tensorflow.org/versions/master/tutorials/image_recognition/index.html

Below is a cut-and-paste from a recent announcement sent to discuss@tensorflow.org.

===========

Dear TensorFlow community,

Today we are releasing our best image classifier trained on ImageNet data. As described in our recent Arxiv preprint at http://arxiv.org/abs/1512.00567, an ensemble of four of these models achieves 3.46% top-5 error on the validation set of the ImageNet whole image ILSVRC2012 classification task (compared with our ensemble from last year that won the 2014 ImageNet classification challenge with a 6.66% top-5 error rate).

In this release, we are supplying code and data files containing the trained model parameters for running the image classifier on:
Both desktop and mobile environments
Employing either a C++ or Python API.

In addition, we are providing a tutorial that describes how to use the image recognition system for a variety of use-cases.
    http://www.tensorflow.org/tutorials/image_recognition/index.html

This release allows one to compute higher-level visual features and/or perform image recognition on the ImageNet 1000 object label set*. We are actively working on refactoring our code so that we can open-source a complete training system for this model so that others can train on their own data or fine-tune it for other purposes (Some custom training operations for this model rely on code that is shared between our non-open-sourced DistBelief system and TensorFlow and we are actively working on disentangling this).

We wish to especially acknowledge Christian Szegedy, Sergey Ioffe and Vincent Vanhoucke for developing this network and helping make this happen.

Happy Holidays-
Sherry, Pete, Chris, Jon with contributions from many team members

* The 1000 object labels are listed at:
http://image-net.org/challenges/LSVRC/2014/browse-synsets
